src/models/tests/test_simple_diabetes_model.py

```diff
@@ -31,6 +31,7 @@ def test_simple_metabolism_model():
 
         (net_change_in_bg, t_5min, carb_amount, insulin_amount, iob_5min,) = smm.run(
             carb_amount=carb_amount, insulin_amount=insulin_amount,
+            five_min=True
         )
 
         assert iob_5min[-1] < (INSULIN_DECAY_8HR_EPSILON * insulin_amount)
```

Comment:
this comment refers to line 42. I am not suggesting we do this now, but I think the correct test for approximately the insulin on board from the scheduled basal rate is to run the simulation over 6-8 hours to get the insulin on board.

Reply:
If I understand correctly, I think the test you're wanting is in the other file in the function `test_simple_metabolism_model_class_iob_steady_state`, which tests the steady state value of the model class against what your original code has. This one was more of a smoke test with super basic assertions, but in looking at the function names maybe they could be made more clear